IBMigrationTool/InterfaceBuilderFileMigrator.swift

```diff
@@ -33,6 +65,9 @@ final class InterfaceBuilderFileMigrator {
     /// represent the fixed version of the file!
     var string: String
     
+    /// String matches for umigrateable text. Map from line number to the string matched.
```

Comment:
Typo maybe un-migratable? Not causing any readability or understanding issues though